Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:2.7.12
+EXPOSE 5000
+VOLUME /user/octoprint/.octoprint
+
+WORKDIR /opt/octoprint
+
+#install ffmpeg
+RUN  wget -O ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-32bit-static.tar.xz \

```

Comment:
Can you checksum the artifact before moving forward? Just to confirm the integrity of the download? See here for an example of this:
https://github.com/docker-library/docker/blob/05678f7f3198d0787d78de9bdc084dcffccf7276/1.12/Dockerfile#L14